runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -1197,14 +1197,8 @@ public synchronized CompletableFuture<Layout> invalidateLayout() {
             List<String> servers = Optional.ofNullable(latestLayout)
                     .map(Layout::getLayoutServers)
                     .orElse(bootstrapLayoutServers);
-            final List<String> responsiveServer =
-                    servers.stream().filter(server -> !latestLayout.getUnresponsiveServers()
-                            .contains(server))
-                            .collect(Collectors.toList());
-            if (responsiveServer.isEmpty()) {
-                throw new IllegalStateException("All servers are unresponsive");
-            }
-            layout = fetchLayout(responsiveServer);
```

Comment:
This will just restore the previous behavior and it is not a fix for the original problem which was reducing the stall time during reconfiguration. The proper solution would be to implement a quorum read, or an easier solution, a broadcast.

Another simpler solution is to reduce the RPC timeout for just that request such that it can discover the new layout faster.

Reply:
I agree, this is a good suggestion. The goal of this PR was just to address the bug seen in smoke environments. I'll address this in a subsequent PR.